syntax = "proto3";
package fixbug;
//定义下面选项，表示生成service服务类和rpc方法描述，默认不生成
option cc_generic_services =true;

//protoc test.proto --cpp_out=./

message ResultCode{
    int32 errorcode=1;
    bytes errormsg=2;
}

message GetFriendListRequest{
    uint32 id=1;
}

message GetFriendListResponse{
    ResultCode result =1;
    repeated bytes friendlist=2;
}

service FriendServiceRpc{
    rpc GetFriendList(GetFriendListRequest) returns(GetFriendListResponse);
}